lib/ripper-tags/parser.rb

```diff
@@ -95,6 +98,16 @@ def on_dyna_symbol(*args)
     end
   end
 
+  def on_array(args)
+    if args && args[0] == :args
+      args[1..-1]
+    end
+  end
```

Comment:
This is the better way of handling arrays, rather than the `on_args` processing I first suggested